src/test/fio/fio_ceph_objectstore.cc

```diff
@@ -329,7 +329,7 @@ int fio_ceph_os_queue(thread_data* td, io_u* u)
   fio_ro_check(td, u);
 
   auto job = static_cast<Job*>(td->io_ops_data);
-  auto& object = job->objects[u->file->engine_data];
+  auto& object = job->objects[(uintptr_t)u->file->engine_data];
```

Comment:
better off using `uint32_t` to be consistent with how it is set. `uintptr_t` is a type for holding a pointer.